### ISSUE TITLE
Can not use shillelagh to query github issues/pulls from Superset

### ISSUE THREAD

**Issue by ivanrezic** [betodealmeida/shillelagh#487]
**Describe the bug**
I was following this blogpost: https://preset.io/blog/accessing-apis-with-superset/ to setup shillelagh in superset.
I have installed `shillelagh[all]==1.3.1`.

**To Reproduce**
1. add new shillelagh db connection in **superset**:
<img width="504" alt="image" src="https://github.com/user-attachments/assets/643bffaa-5a96-43b9-bb48-d66d800acd8a">

2. Add engine parameters
<img width="503" alt="image" src="https://github.com/user-attachments/assets/7f3b4271-c0cb-4501-91d4-be2975334c73">

3. click connect: 
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/beffcda3-b725-4b1e-9ba1-a66b4fb4868e">

Error message:
```
An error occurred while creating databases: (builtins.NoneType) None
[SQL: 'apsw.Connection' object has no attribute 'create_scalar_function']
(Background on this error at: https://sqlalche.me/e/14/dbapi)
```

I have also imported db downloaded from [mentioned blog post](https://preset.io/blog/accessing-apis-with-superset/#:~:text=Alternatively%2C%20you%20can%20simply%20import%20this%20database%20into%20your%20workspace.) and when imported it was displayed in db connections but when I try to query in SQL Lab I get this error:

<img width="1375" alt="image" src="https://github.com/user-attachments/assets/70bbce64-eeeb-4dbd-9ba9-4e7db3c44236">

**Expected behavior**
Querying works and connection is possible.

**Desktop (please complete the following information):**
- Superset run via helm charts, docker image version 4.0.1

**Comment by nielstenboom** [comment]
Got it working, but needed to explicitly set the following in the Superset requirements

```
apsw==3.46.*
```

Could it be the dependencies of this project need to be defined a bit stricter for the latest release?

**Comment by ivanrezic** [comment]
Works, thanks!

**Comment by betodealmeida** [comment]
Thanks for the report, @ivanrezic, and for the fix, @nielstenboom! I'll update the dependency version.

**Comment by betodealmeida** [comment]
I released [1.3.2](https://github.com/betodealmeida/shillelagh/releases/tag/1.3.2) with a bumped version, should be fine now.
